app/src/main/java/ru/thstdio/aa2020/ui/detail/MoviesDetailsScreen.kt

```diff
@@ -0,0 +1,9 @@
+package ru.thstdio.aa2020.ui.detail
+
+import com.github.terrakok.cicerone.androidx.FragmentScreen
+
+object MoviesDetailsScreen {
```

Comment:
what sense of object here?
maybe just
class MoviesDetailsScreen : FragmentScreen